pkg/awsutil/session.go

```diff
@@ -47,6 +51,7 @@ type Credentials struct {
 
 	CustomEndpoints config.CustomEndpoints
 	session         *session.Session
```

Comment:
one question not impacting the migration: we replace `DefaultRegionID`/`DefaultAWSPartitionID` and other API in batch.go with go v2's value and code, could we directly remove `session` field here considering it is v1 only?